CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 4.5.3
+- Bump lodash version to 4.17.21 ([#x](https://github.com/bigcommerce/paper-handlebars/pull/x))
```

Comment:
Should this have a pull request number instead of `x` in the URL?

Reply:
nice catch!